frontend/uttarakhand-floods-2013/style.css

```
@@ -0,0 +1,1327 @@
+
+.wave-1 {
+    fill: rgba(56, 189, 248, 0.16);
+    animation: waveDrift 11s linear infinite;
+}
+
+.wave-2 {
+    fill: rgba(3, 105, 161, 0.22);
+    animation: waveDrift 17s linear infinite reverse;
+}
+
+@keyframes waveDrift {
```

Stylelint (17.14.0)

[error] Expected keyframe name "waveDrift" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)